### PR TITLE
Adds Pedal Fingerprints for Honda and small change to Pilot / Ridgeline steerKpV and steerKiV to fit 3.3333 ratio

### PR DESCRIPTION
**Pull request by jfrux**
This is based on Joel's Pedal PR but should contain changes from  #251 `0.4.7.1` release.
- [x] Updates Acura RDX Fingerprint Changes to Support Comma Pedal
- [x] Updates 2017 Honda Pilot Touring & 2016 Honda Pilot EX-L Fingerprint Changes to Support Comma Pedal
- [x] Updates Pilot & Ridgeline steerKpV and steerKiV to match the 3.3333 ratio of the other Honda vehicles.
- [x] Adds comments to `selfdrive/car/honda/values.py` to explain the Pedal changes.
- [x] Get response on questions from @rbiasini relating to comment formatting, and fingerprint formats.
- [x] Verifying the steerKpV and steerKiV changes I made on my drive home today.
- [ ] Fix issues with minimum steering speeds not showing steering warnings.  Mine is the Pilot and it's not showing the warning anymore below 12mph.

### CODE REVIEW

**Comment by rbiasini** [comment]
@jfrux looks good. Can you please add a comment that 512 and 513 are reserved comma pedal? Also, I would prefer to leave the toyota changes out for now, as pedal isn't supported for toyota vehicles yet.
Other than that I verified that the changes are as intended, so it's good to go.

**Comment by jfrux** [comment]
@rbiasini Do you want the comments + the 512, 513's to be on a new line similar to how the sent messages were before?

Example:
```python
...
CAR.PILOT: [{
  57L: 3, 145L: 8, 228L: 5, 229L: 4, 308L: 5, 316L: 8, 334L: 8, 339L: 7, 342L: 6, 344L: 8, 379L: 8, 380L: 8, 392L: 6, 399L: 7, 419L: 8, 420L: 8, 422L: 8, 425L: 8, 426L: 8, 427L: 3, 432L: 7, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 538L: 3, 542L: 7, 545L: 5, 546L: 3, 597L: 8, 660L: 8, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 808L: 8, 819L: 7, 821L: 5, 829L: 5, 837L: 5, 856L: 7, 871L: 8, 882L: 2, 884L: 7, 891L: 8, 892L: 8, 923L: 2, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1027L: 5, 1029L: 8, 1036L: 8, 1039L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1108L: 8, 1125L: 8, 1296L: 8, 1424L: 5, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1616L: 5, 1618L: 5, 1668L: 5,
  # Comma Pedal Support
  512L: 6, 513L: 6, 
}],
...
```

**Comment by jfrux** [comment]
Another question, should I add logic for Pedal support and `stop_and_go` in `interface.py`
You would think that cars with Pedal would need to have different settings in some areas?

**Comment by jfrux** [comment]
@rbiasini Actually, another question (sorry) but should the Finger print with Pedal be in addition to the existing fingerprint for all vehicles? So the fingerprint matches with pedal and without pedal?

Example:
```python
# 2017 Pilot Touring AND 2016 Pilot EX-L
CAR.PILOT: [
# w/o Pedal Interceptor
{
  57L: 3, 145L: 8, 228L: 5, 229L: 4, 308L: 5, 316L: 8, 334L: 8, 339L: 7, 342L: 6, 344L: 8, 379L: 8, 380L: 8, 392L: 6, 399L: 7, 419L: 8, 420L: 8, 422L: 8, 425L: 8, 426L: 8, 427L: 3, 432L: 7, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 538L: 3, 542L: 7, 545L: 5, 546L: 3, 597L: 8, 660L: 8, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 808L: 8, 819L: 7, 821L: 5, 829L: 5, 837L: 5, 856L: 7, 871L: 8, 882L: 2, 884L: 7, 891L: 8, 892L: 8, 923L: 2, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1027L: 5, 1029L: 8, 1036L: 8, 1039L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1108L: 8, 1125L: 8, 1296L: 8, 1424L: 5, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1616L: 5, 1618L: 5, 1668L: 5
},
# w/ Pedal Interceptor (adds 512L & 513L)
{
  57L: 3, 145L: 8, 228L: 5, 229L: 4, 308L: 5, 316L: 8, 334L: 8, 339L: 7, 342L: 6, 344L: 8, 379L: 8, 380L: 8, 392L: 6, 399L: 7, 419L: 8, 420L: 8, 422L: 8, 425L: 8, 426L: 8, 427L: 3, 432L: 7, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 512L: 6, 513L: 6, 538L: 3, 542L: 7, 545L: 5, 546L: 3, 597L: 8, 660L: 8, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 808L: 8, 819L: 7, 821L: 5, 829L: 5, 837L: 5, 856L: 7, 871L: 8, 882L: 2, 884L: 7, 891L: 8, 892L: 8, 923L: 2, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1027L: 5, 1029L: 8, 1036L: 8, 1039L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1108L: 8, 1125L: 8, 1296L: 8, 1424L: 5, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1616L: 5, 1618L: 5, 1668L: 5
}],
```

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -70,18 +71,20 @@ class CAR:
...
   },
-  # Odyssey Elite
+  # 2018 Odyssey Elite w/ Added Comma Pedal Support (512L & 513L)
   {
     57L: 3, 148L: 8, 228L: 5, 229L: 4, 304L: 8, 342L: 6, 344L: 8, 380L: 8, 399L: 7, 411L: 5, 419L: 8, 420L: 8, 427L: 3, 432L: 7, 440L: 8, 450L: 8, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 512L: 6, 513L: 6, 542L: 7, 545L: 6, 597L: 8, 662L: 4, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 806L: 8, 808L: 8, 817L: 4, 819L: 7, 821L: 5, 825L: 4, 829L: 5, 837L: 5, 856L: 7, 862L: 8, 871L: 8, 881L: 8, 882L: 4, 884L: 8, 891L: 8, 892L: 8, 905L: 8, 923L: 2, 927L: 8, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1029L: 8, 1036L: 8, 1052L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1092L: 1, 1108L: 8, 1110L: 8, 1125L: 8, 1296L: 8, 1302L: 8, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1614L: 5, 1616L: 5, 1619L: 5, 1623L: 5, 1668L: 5
   }],
-  # Includes 2017 Touring and 2016 EX-L messaging.
+  # 2017 Pilot Touring AND 2016 Pilot EX-L w/ Added Comma Pedal Support (512L & 513L)
```
Comment (jfrux) [inline selfdrive/car/honda/values.py]: @rbiasini  Should comments / fingerprints be formatted like this instead with separate fingerprints, one for pilot with pedal, one without pedal?

```python
# 2017 Pilot Touring AND 2016 Pilot EX-L
CAR.PILOT: [
# w/o Pedal Interceptor
{
  57L: 3, 145L: 8, 228L: 5, 229L: 4, 308L: 5, 316L: 8, 334L: 8, 339L: 7, 342L: 6, 344L: 8, 379L: 8, 380L: 8, 392L: 6, 399L: 7, 419L: 8, 420L: 8, 422L: 8, 425L: 8, 426L: 8, 427L: 3, 432L: 7, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 538L: 3, 542L: 7, 545L: 5, 546L: 3, 597L: 8, 660L: 8, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 808L: 8, 819L: 7, 821L: 5, 829L: 5, 837L: 5, 856L: 7, 871L: 8, 882L: 2, 884L: 7, 891L: 8, 892L: 8, 923L: 2, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1027L: 5, 1029L: 8, 1036L: 8, 1039L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1108L: 8, 1125L: 8, 1296L: 8, 1424L: 5, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1616L: 5, 1618L: 5, 1668L: 5
},
# w/ Pedal Interceptor (adds 512L & 513L)
{
  57L: 3, 145L: 8, 228L: 5, 229L: 4, 308L: 5, 316L: 8, 334L: 8, 339L: 7, 342L: 6, 344L: 8, 379L: 8, 380L: 8, 392L: 6, 399L: 7, 419L: 8, 420L: 8, 422L: 8, 425L: 8, 426L: 8, 427L: 3, 432L: 7, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 512L: 6, 513L: 6, 538L: 3, 542L: 7, 545L: 5, 546L: 3, 597L: 8, 660L: 8, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 808L: 8, 819L: 7, 821L: 5, 829L: 5, 837L: 5, 856L: 7, 871L: 8, 882L: 2, 884L: 7, 891L: 8, 892L: 8, 923L: 2, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1027L: 5, 1029L: 8, 1036L: 8, 1039L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1108L: 8, 1125L: 8, 1296L: 8, 1424L: 5, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1616L: 5, 1618L: 5, 1668L: 5
}],
```

Comment (jfrux) [reply selfdrive/car/honda/values.py]: @vanillagorillaa answered my question on this comment.

**Comment by jfrux** [comment]
Okay so we still need to warn steering on cars that have a cut off on steering at the ECU.
The pilot has a cut off on steering at 12mph so I need a warning to throw.
It appears that something changed in 0.4.6/0.4.7 that has caused this experience to go away.

It now just shows as engaged always down to zero.

**Pull request closed without merging**